aiidalab_widgets_base/computational_resources.py

```diff
@@ -477,8 +484,12 @@ def _on_setup_ssh_button_pressed(self, _=None):
 
             # Write private key in ~/.ssh/ and use the name of upload file,
             # if exist, generate random string and append to filename then override current name.
+            # TODO(danielhollas): I don't think this works as intended. If there is an existing private key,
+            # the new filename is never propagated to the caller here.
             self._add_private_key(private_key_abs_fname, private_key_content)
 
+        # TODO(danielhollas): I am not sure this is correct. What if the user wants
+        # to overwrite the private key? The configuration would never be written.
```

Comment:
Given that the primary focus of the PR isn't on this topic, could we move this information to an issue instead? It seems unnecessary to have multiple locations where information about potential problems can be accessed.

Reply:
I've opened #516 to track these issues. However, I would like to keep the TODOs in the code. I think it is quite important because if somebody wanders into this code later, they shouldn't need to know about every open issue on Github. I've added the link to the GH issue so that these things are crosslinked.